backend/src/main/java/com/crm/dto/request/NewAccountRequest.java

```diff
@@ -27,16 +26,15 @@
 @Getter
 public class NewAccountRequest {
 
-    @NotNull(message = LOGIN_CAN_NOT_BE_EMPTY)
     @NotBlank(message = LOGIN_CAN_NOT_BE_EMPTY)
     @Size(max = 50, message = LOGIN_LENGTH_CAN_NOT_BE_GREATER_THAN)
     private String login;
-    @NotNull(message = PASSWORD_CAN_NOT_BE_EMPTY)
+
     @NotBlank(message = PASSWORD_CAN_NOT_BE_EMPTY)
     @Size(min = 8, max = 30, message = PASSWORD_LENGTH_MUST_BETWEEN)
     private String password;
+
     @Email(message = EMAIL_FORMAT_INVALID)
-    @NotNull
```

Comment:
I think this annotation should be left for the email.

Reply:
sorry, I didn't get your point

Reply:
I mean that you deleted NotNull but not sure why? I think it was good before

Reply:
Sorry, I was wrong. I think this is ok.